### ISSUE TITLE
Unable to use Baum VarioPro 80 with NVDA

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#6218]
I've been able to use a VarioPro 80 from Baum, which doesn't seem to be recognised by NVDA.

```
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._handleResponse (10:49:38):
Unknown command '@', arg ''
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._onReceive (10:49:38):
Ignoring byte before escape: '\x14'
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._handleResponse (10:49:38):
Unknown command '@', arg ''
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._onReceive (10:49:38):
Ignoring byte before escape: '\x14'
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._handleResponse (10:49:38):
Unknown command '@', arg ''
DEBUGWARNING - brailleDisplayDrivers.baum.BrailleDisplayDriver._onReceive (10:49:38):
Ignoring byte before escape: '\x14'
ERROR - braille.BrailleHandler.setDisplayByName (10:49:39):
Error initializing display driver
Traceback (most recent call last):
  File "braille.pyc", line 1475, in setDisplayByName
  File "brailleDisplayDrivers\baum.pyc", line 200, in __init__
RuntimeError: No Baum display found
```

Other screen readers seem to work fine.

**Comment by jcsteh** [comment]
Duplicate of #5162. See the last comment on that issue for details.
